pkg/interactive-installer/run-installer.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
I am curious why we do all of this in shell - which then requires `jq` and other dependencies - when all of the capabilities are in the rust-compiled program anyways. Doesn't this just act as a wrapper to `installer`? Why not have it all there?

Reply:
Correct, this is a first step before the whole thing. The plan is to have everything in the installer.